### PR TITLE
Feature request: Adding advertisement screen

### LINKED ISSUE #993: Bug Report: Missing REACT_APP_BACKEND_WEBSOCKET_URL=ws://localhost:4000/graphql
**Describe the bug**

To use the web sockets we need *REACT_APP_BACKEND_WEBSOCKET_URLl** as an env variable. It is not present in .env.example
**To Reproduce**

1. Socket server is not connected 

**Expected behavior**
CONNECT TO SOCKET SERVER

### LINKED ISSUE #996: Feature Request: Adding advertisement management screen for the admin
**Is your feature request related to a problem? Please describe.**
Advertisement management window should be present using which the admin can create new advertisement that were shown to end users in their interface in the form of pop-up or banners. The ads would be managed completely by the admin while the Tarawa app only make sure to show them to the user depending on the time that is decided by the admin.

**Describe the solution you'd like**
Admin interface with such features would be beneficial

**Describe alternatives you've considered**
I have not considered any other alternatives
**Approach to be followed (optional)**
Creating an admin user interface giving it to tabs the active campaigns and the archived campaigns. The active campaigns contains the list of advertisements in the car currently running in that organisation while the archived campaigns contains the list of the advertisements that were running previously, which means the end date of those advertisement is over.

**Additional context**

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### LINKED ISSUE #1071: Remove Add plugin dialog
**Describe the bug**
Add plugin dialog is not longer required as plugins are by default loaded from the API. Refer to YouTube channel to details.

**Screenshots**
Removing this 
Screenshot 2023-11-17 at 9 35 33 AM

Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by SiddheshKukade**
**What kind of change does this PR introduce?**
In my previous PR   #976 of websockets I forgot to add this env.example file. 

Added Advertisement User interface:

 Summary : 
- Developed User interface for advetisement store only loading ads matching with the current logged in org of user with features like deleting seeing the ongoing ad campgins,etc.
- Developed a feature in Talawa User Web app to showcase those advertisement as promoted posts
**Issue Number:**

Fixes #993
Fixes #996 
Fixes #1071 

**Snapshots/Videos:**
Video Demo :  https://www.youtube.com/watch?v=66q8GHktN8A

[IMAGE ALT TEXT HERE](https://www.youtube.com/watch?v=66q8GHktN8A)

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

**Comment by SiddheshKukade** [comment]
@palisadoes sir, please merge this is a very short fix.

**Review by palisadoes — APPROVED** [review]

**Review by tasneemkoushar — DISMISSED** [review]

**Comment by SiddheshKukade** [comment]
I've added the further code regarding advertisements. Please don't merge this. I'll let you guys know once it's ready.

**Review by palisadoes — APPROVED** [review]

**Comment by palisadoes** [comment]
Please fix the conflicting files

**Review by tasneemkoushar — APPROVED** [review]

**Comment by tasneemkoushar** [comment]
@SiddheshKukade code changes seem good to me, but try to fix the tests which are failing.

File: src/screens/UserPortal/Home/Home.tsx
Comment (SiddheshKukade) [reply src/screens/UserPortal/Home/Home.tsx]: Resolved

**Comment by SiddheshKukade** [comment]
@tasneemkoushar yes, I've fixed the failing tests and conflicts. Now it is good to be merged.

**Comment by SiddheshKukade** [comment]
@palisadoes sir I ran the same command on my machine. I'm not getting any errors. 
I think it's okay to merge it.
Screenshot 2023-10-27 at 3 33 42 PM

**Comment by palisadoes** [comment]
@SiddheshKukade The code coverage has dropped dramatically. Please resolve this.

**Comment by SiddheshKukade** [comment]
Working on it !

**Comment by palisadoes** [comment]
@SiddheshKukade Are you still working on this? Should it be closed?

**Comment by SiddheshKukade** [comment]
Working on it

**Comment by SiddheshKukade** [comment]
@palisadoes Sir, I've aded all tests for my new components. I think this PR is good to be merged. 

//cc @tasneemkoushar  @DMills27

**Review by DMills27 — APPROVED** [review]

**Comment by palisadoes** [comment]
@SiddheshKukade our code coverage will fall to 90% after this PR is merged. How do you propose to resolve this?

**Comment by SiddheshKukade** [comment]
Hello sir, I've actually added only two components here and also the test
files for both of them. I'm unsure why it's drastically fallen by 4%.

**Comment by palisadoes** [comment]
@SiddheshKukade 

Please add your code coverage report to the PR.

I've been trying to update the one auto-generated by the PR without success

**Review by palisadoes — APPROVED** [review]

**Pull request merged**
